src/main/java/ru/arturios/Commands.java

```diff
@@ -0,0 +1,67 @@
+package ru.arturios;
+
+import org.springframework.stereotype.Component;
+import ru.arturios.products.ProductRepository;
+
+@Component
+public class Commands {
+
+    private final ProductRepository productRepository;
+    private final Cart cart;
+
+    public Commands(ProductRepository productRepository, Cart cart) {
+        this.productRepository = productRepository;
+        this.cart = cart;
+    }
+
+    public void doAuth(String s) {
+        if (s.startsWith("INFO")) {
+            System.out.println("****************************");
+            System.out.println("**      Набор команд      **");
+            System.out.println("**  add - добавить товар  **");
+            System.out.println("** del - удалить товар    **");
+            System.out.println("** all - товары в корзине **");
+            System.out.println("****************************");
+
+        } else if (s.startsWith("all")){
+            System.out.println("Всего товаров:");
+            for (int i = 1; i <= productRepository.getAll().size(); i++) {
+                System.out.printf("id: %s Имя: %s Цена: %s\n",productRepository.getById(String.valueOf(i)).getId(),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.